domains/inklusion.is-local.org.json

```diff
@@ -0,0 +1,21 @@
+{
+    "description": "A university project on inclusion in relation to the realization of transitions out of the school system",
+
+    "domain": "is-local.org",
+    "subdomain": "inklusion",
+
+    "owner": {
+        "repo": "https://github.com/strecke/register",
+        "email": "inklusion-info@proton.me"
+    },
+
+    "record": {
+        "A": "128.204.223.70",
+        "CNAME": "inklusion.is-local.org",
```

Comment:
If you wanted CNAME records, you must make sure that there are no other records present.